src/openrct2/localisation/Localisation.h

```diff
@@ -112,11 +112,16 @@ class Formatter
         return CurrentBuf;
     }
 
-    void Increment(std::size_t count)
+    void Increment(size_t count)
```

Comment:
Why dropping the `std`?

Reply:
Where else do we have `std::size_t`? I think it is just making things consistent.

Reply:
I temporarily changed it to a `ssize_t` before opting for another solution. Could be changed back.